src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -613,19 +613,55 @@ groupContainer viewSegment highlightables =
                             [ Css.backgroundColor Css.transparent
                             , Css.Global.children
                                 [ Css.Global.selector ":first-child"
```

Comment:
Right now, the before and after elements are added to the first and last span children of the mark. 

<img width="505" alt="Screen Shot of the DOM highlighting the structure" src="https://user-images.githubusercontent.com/8811312/193141730-23b9e611-4add-4177-9d69-24ef8277f0d1.png">

I'm wondering if instead, we should have the pseudo-elements belong to the mark itself? Conceptually, it seems like that's where it belongs to me 🤔 I think I added the before element to the first child for the high-contrast mode for visual reasons, rather than for semantic reasons

Reply:
Ahh okay! 👍